src/map_object_move.c

```diff
@@ -987,32 +988,32 @@ void sub_02064464 (MapObject * mapObj)
 }
 
 static const int Unk_020EE79C[] = {
-    0x1,
-    0x0,
-    0x3,
-    0x2
+    DIR_SOUTH,
+    DIR_NORTH,
+    DIR_EAST,
+    DIR_WEST
 };
 
-int sub_0206447C (int param0)
+int Direction_GetOpposite (int dir)
```

Comment:
`Direction_Flip`? Or maybe just `FlipDirection`?

Reply:
Opposite is fine tbh